src/assets/scss/pages/story.scss

```diff
@@ -1,11 +1,36 @@
 .author-information {
+  align-items: center;
   display: flex;
   flex-direction: row;
-  margin-left: 3em;
+  flex-wrap: wrap;
+  justify-content: flex-start;
   margin-top: 0.5em;
 
+  & > * {
+    margin-right: 4rem;
+  }
+
+  .name,
+  .target-product {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
```

Comment:
you should be able to remove this prop since the default value for flex-direction is row.

Reply:
Done